pages/profil.tsx

```diff
@@ -1,59 +1,104 @@
 import { withTransaction } from '@elastic/apm-rum-react'
 import { GetServerSideProps } from 'next'
-import React from 'react'
+import React, { ChangeEvent } from 'react'
 
-import { Conseiller, UserStructure } from 'interfaces/conseiller'
+import { Switch } from 'components/ui/Switch'
+import { UserStructure } from 'interfaces/conseiller'
 import { ConseillerService } from 'services/conseiller.service'
 import styles from 'styles/components/Layouts.module.css'
 import useMatomo from 'utils/analytics/useMatomo'
+import useSession from 'utils/auth/useSession'
 import { withMandatorySessionOrRedirect } from 'utils/auth/withMandatorySessionOrRedirect'
-import withDependance from 'utils/injectionDependances/withDependance'
+import { useConseiller } from 'utils/conseiller/conseillerContext'
+import { useDependance } from 'utils/injectionDependances'
 
 interface ProfilProps {
-  conseiller: Conseiller
   structureConseiller: string
   pageTitle: string
 }
 
-function Profil({ conseiller, structureConseiller }: ProfilProps) {
+function Profil({ structureConseiller }: ProfilProps) {
   const labelAgence =
     structureConseiller === UserStructure.MILO ? 'Mission locale' : 'agence'
 
+  const conseillerService =
+    useDependance<ConseillerService>('conseillerService')
+
+  const [conseiller, setConseiller] = useConseiller()
+  const { data: session } = useSession<true>({ required: true })
+
+  function toggleNotificationsSonores(e: ChangeEvent<HTMLInputElement>) {
```

Comment:
nitpick
```suggestion
  function toggleNotificationSonore(e: ChangeEvent<HTMLInputElement>) {
```
pourquoi au pluriel?

Reply:
Alors pour le coup, c'est @jozolr qui a fait l'API au pluriel, et en conséquence on a gardé ce pluriel. Je dirais de ma compréhension pour "a activé les notifications sonores". Par contre, je réalise que dans le code ailleurs, on l'a mis au singulier. Du coup, je te propose dans un premier temps de rester cohérent et de tout mettre au pluriel. Et demain à ton retour de conf, si tu souhaites plutôt le singulier, on fait le rename une fois la branche mergée.